chef/cookbooks/crowbar/recipes/crowbar-upgrade.rb

```diff
@@ -47,13 +47,14 @@
 
   bash "disable_openstack_services" do
     code <<-EOF
-      for i in $(systemctl list-units openstack* --no-legend | cut -d" " -f1) \
+      for i in $(systemctl list-units openstack* --no-legend | cut -d" " -f1 | grep -v keystone | grep -v neutron) \
                drbd.service \
                pacemaker.service;
       do
         systemctl disable $i
       done
     EOF
+    not_if { ::File.exist?("/usr/sbin/crm") }
```

Comment:
> pacemaker should not be happy by something external disabling services.

Huh?  The opposite is true.  Services controlled by pacemaker must not be enabled, because they must not be started on boot by systemd, but instead get started by Pacemaker.  And indeed [we already make Chef do this](https://github.com/crowbar/crowbar-ha/blob/master/chef/cookbooks/crowbar-pacemaker/providers/service.rb#L41).  So in the HA context, this code should be a no-op, because the services will already be disabled.  Maybe you are confusing disable with stop?